.gitpod.yml

```diff
@@ -6,7 +6,6 @@
 
 tasks:
   - init: npm install && npm run build
```

Comment:
```suggestion
  - init: npm install && npm run package
```

So that folks that just want to try out the PR/extension will have a ready-made one.